job-dsl-plugin/src/main/groovy/javaposse/jobdsl/plugin/WarnConfigChangeAction.java

```diff
@@ -0,0 +1,53 @@
+package javaposse.jobdsl.plugin;
+
+import hudson.Util;
+import hudson.model.AbstractItem;
+import hudson.model.Action;
+
+import java.io.IOException;
+import java.util.logging.Logger;
+
+public class WarnConfigChangeAction implements Action {
```

Comment:
This can be merged with SeedJobAction. I think we don't need a separate action for this. Either both are used or none.